src/plugins/workspace/public/components/service_card/workspace_list_card.tsx

```diff
@@ -88,62 +86,63 @@ export const WorkspaceListCard = (props: WorkspaceListCardProps) => {
   const isDashboardAdmin = application.capabilities.dashboards?.isDashboardAdmin;
 
   return (
-    <EuiPanel paddingSize="s" hasBorder={false} hasShadow={false}>
-      <EuiFlexGroup gutterSize="xs" alignItems="center">
-        <EuiFlexItem grow={3}>
-          <EuiTitle>
-            <h4>Workspaces</h4>
-          </EuiTitle>
-        </EuiFlexItem>
-        <EuiFlexItem grow={1}>
-          <EuiToolTip position="bottom" content={WORKSPACE_LIST_CARD_DESCRIPTION}>
-            <EuiIcon type="iInCircle" aria-label="workspace list card description" />
-          </EuiToolTip>
-        </EuiFlexItem>
-        <EuiFlexItem grow={5}>
-          <EuiSelect
-            value={filter}
-            data-test-subj="workspace_filter"
-            onChange={(e) => {
-              setFilter(e.target.value);
-            }}
-            options={[
-              {
-                value: 'viewed',
-                text: i18n.translate('workspace.list.card.filter.viewed', {
-                  defaultMessage: 'Recently viewed',
-                }),
-              },
-              {
-                value: 'updated',
-                text: i18n.translate('workspace.list.card.filter.updated', {
-                  defaultMessage: 'Recently updated',
-                }),
-              },
-            ]}
-          />
-        </EuiFlexItem>
-        {isDashboardAdmin && (
-          <EuiFlexItem grow={false}>
-            <EuiToolTip position="top" content="Create workspace">
-              <EuiButtonIcon
-                data-test-subj="create_workspace"
-                aria-label="create workspace"
-                display="base"
-                iconType="plus"
-                size="m"
-                onClick={() => {
-                  application.navigateToApp(WORKSPACE_CREATE_APP_ID);
-                }}
-              />
+    <EuiCard
+      textAlign="left"
```

Comment:
please remove border and shadow for the card `hasBorder={false} hasShadow={false}`

Reply:
Thanks, already updated